### PR TITLE
adding initial account sections

### PR DESCRIPTION
**Pull request by davidsmejia**
## Issue Number

#290 

## Purpose/Implementation Notes

Adds the account sections first pass, needs to tie into the API and also move some logic into their own components still

## Types of changes

- New feature (non-breaking change which adds functionality)

## Functional tests

Runs Locally

## Checklist

- [x] Lint and unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: client/src/api/index.js [hunk 1]
```diff
@@ -4,6 +4,13 @@ export const host = process.env.API_HOST
 export const version = process.env.API_VERSION
 export const path = `${host}/${version}/`
 
+// create new form data object and attack keys from object
```
Comment (Philinphiladelphia) [inline client/src/api/index.js:7]:
```suggestion
// create new form data object and attach keys from object
```

Comment (davidsmejia) [reply client/src/api/index.js:7]: i prefer it my way

File: client/src/api/index.js [hunk 2]
```diff
@@ -112,5 +148,48 @@ export default {
 
       return [tokenRequest, userRequest]
     }
+  },
+  teams: {
```
Comment (Philinphiladelphia) [inline client/src/api/index.js:152]: This language is a bit strange. I think they're called "organizations" throughout the backend at least- are we calling them teams on the frontend?

Comment (davidsmejia) [reply client/src/api/index.js:152]: yeah i wanted the api to reflect what the front end is saying so it is easier to keep straight

File: client/src/components/resources/ManageResourceCard.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+import { ResourceCard } from './ResourceCard'
+
+export const ManageResourceCard = ({ resource }) => {
+  // How do we get pending requests
+  // What do we link to when there are pending requests
+  const editLink = `/account/manage-resources/${resource.id}`
+  const teamLink = `/account/teams/${resource.organization.id}`
+  const viewLink = `/resources/${resource.id}`
+
+  // modal for deleting a resource?
```
Comment (Philinphiladelphia) [inline client/src/components/resources/ManageResourceCard.js:13]: Could you make issues out of this and the above comment? It's a bit hard to keep track of in this form.

File: client/src/pages/account/teams/index.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { DrillDownNav } from 'components/DrillDownNav'
+import { AccountEmptyPage } from 'components/AccountEmptyPage'
+import EmptyTeams from '../../../images/empty-teams.svg'
+
+const Teams = () => {
```
Comment (Philinphiladelphia) [inline client/src/pages/account/teams/index.js:6]: Same question here with teams.

File: client/src/components/resources/ManageResourceCard.js (hunk 3 above)
Comment (davidsmejia) [reply client/src/components/resources/ManageResourceCard.js:13]: Thats ok as I flush this out I will end up making that modal

File: client/src/pages/account/teams/index.js (hunk 4 above)
Comment (davidsmejia) [reply client/src/pages/account/teams/index.js:6]: this is consistent with the name of the url

**Pull request merged**